### PR TITLE
create card ricardo-perez

### PR DESCRIPTION
**Pull request by RicardoJosuePerezAltamirano**
# #WeAreDonet Creator Submission Guidelines

* [x] Check all points before submitting the PR.

* [x] By submitting this pull request to add my creator card, I verify that I adhere to the #WeAreDotnet creator card submission guidelines.

* [x] Make sure to follow our [Code of Conduct](https://github.com/wearedotnet/dotnet-creator-cards/blob/master/CODE_OF_CONDUCT.md) 

* [x] We do not accept submissions with the sole intention of promoting a product or company.

* [x]  We add you as a creator, not your company. 

* [x]  The moderator team decides whether we accept or reject a submission.

## When Do We Accept Someone's Creator Card Submission? 

* [x] You must have published at least three content pieces (long form content) since 2022 on any of your channels. 
* .NET itself
* .NET libraries and projects
* .NET languages C#, F#, Visual Basic
* Tools needed during development (for .NET)
* CI/CD
* Only in combination with .NET: SPA Frontend technologies like Angular, Vue.js, React etc. 
* Cloud services related to .NET applications (AWS, Azure...)

* [x] There must be three content pieces in English. 

* [x] There needs to be more than tweets. 

* [x] Longer-form article on LinkedIn is good, but a self-hosted blog or one on a well-known platform like Hashnode, medium etc., is better. 

* [x] Alternatively, you contributed with another form on a platform like YouTube, Twitch, Udemy or Pluralsight. If you have some recorded talks of conferences or anywhere else, that counts too! So in the form of a recorded video or a book. 

## RSS or Atom Feed 

* [x] The content with feed.type = dotnet must be .NET related:
* .NET itself
* .NET libraries and projects
* .NET languages C#, F#, Visual Basic
* Tools needed during development (for .NET)
* CI/CD
* Only in combination with .NET: SPA Frontend technologies like Angular, Vue.js, React etc. 
* Cloud services related to .NET applications (AWS, Azure...)

* [x] the content under feed.type = personal may contain off-topic posts.
* Project management
* leadership
* Other

* [x] Submit an Atom or RSS URL inside your creator card that belongs to your blog 

* [x] If you do not submit an RSS or Atom URL. We cannot consider your feed for the aggregation feature. 

## Content Guideline 

* [x] If you have a self-hosted blog, please ensure you have a valid HTTPS certificate. 

* [x] We host NO malicious or offensive content on the blog (including photos, swearing, etc.)

* [x] You must add the full URL to your GitHub profile to your card, so we can assign it to the member account that used the same GitHub account for sign-up.

### CODE REVIEW

File: creator-cards/ricardo-perez.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Ricardo Josue Perez Altamirano",
+  "type": "CREATOR",
+  "tags": ".NET, C#, ASP.NET, I LOVE DOTNET,Azure",
```
Comment (matthiasjost) [inline creator-cards/ricardo-perez.json:4]: Could you remove I LOVE DOTNET is was only sample data and not meant as a real tag

File: creator-cards/ricardo-perez.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  "channels": [
+    {
+      "name": "Youtube",
+      "url": "https://www.youtube.com/c/RicardoJosue"
+    },{
+      "name": "Sessionize",
+      "url": "https://sessionize.com/ricardo-josue-perez-altamirano/"
+    },{
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/ricardojosue/"
```
Comment (matthiasjost) [inline creator-cards/ricardo-perez.json:15]: @DeeJayTC Can LinkedIn and YouTube be added here as well, or should it be only in the sign-up form?

File: creator-cards/ricardo-perez.json (hunk 1 above)
Comment (RicardoJosuePerezAltamirano) [reply creator-cards/ricardo-perez.json]: yes of couse, the changes is in commit c3584742fd901d3adafc53afc7b136eb96576509

File: creator-cards/ricardo-perez.json (hunk 2 above)
Comment (RicardoJosuePerezAltamirano) [reply creator-cards/ricardo-perez.json:15]: I set this info in the sign up form, and I set here for better contact but if not it is the place for this data i will remove

File: creator-cards/ricardo-perez.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    }
+  ],
+  "feeds": [{
+    "type": "dotnet",
+    "url": "https://www.youtube.com/feeds/videos.xml?channel_id=UC5Rh7fFBjc8_EBEeEXg67aw"
+  }],
+  "projects": [
+    {
+      "name": "DemoServiciosNativosMAUI",
+      "description": "Demo del consumo de servicios nativos en android usando .NET MAUI",
```
Comment (DeeJayTC) [inline creator-cards/ricardo-perez.json:31]: Can you please give us an english description?

File: creator-cards/ricardo-perez.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      "url": "https://github.com/RicardoJosuePerezAltamirano/DemoServiciosNativosMAUI"
+    },
+     {
+      "name": "Sysne.BottomDrawer",
+      "description": "Adaptacion de control Button Drawer para aplicaciones con Xamarin Forms",
```
Comment (DeeJayTC) [inline creator-cards/ricardo-perez.json:36]: See above

File: creator-cards/ricardo-perez.json (hunk 3 above)
Comment (RicardoJosuePerezAltamirano) [reply creator-cards/ricardo-perez.json]: ready in commit 37fa261a5b8ee1d10e89d6dad442afa3cf3342e5

File: creator-cards/ricardo-perez.json (hunk 4 above)
Comment (RicardoJosuePerezAltamirano) [reply creator-cards/ricardo-perez.json]: Ready in commit 37fa261a5b8ee1d10e89d6dad442afa3cf3342e5

File: creator-cards/ricardo-perez.json (hunk 2 above)
Comment (DeeJayTC) [reply creator-cards/ricardo-perez.json:15]: Sorry, this got lost as i apparently accidently resolved the comment, we don't need all this as channels as the content on the page would be heavily duplicated. You can add social infos via the page itself, can you remove the channels here and we merge your feed?

Comment (RicardoJosuePerezAltamirano) [reply creator-cards/ricardo-perez.json]: Ready in commit 7fc2d21933509928a5d67fdd14433949083dcc37

**Comment by DeeJayTC** [comment]
Hey @RicardoJosuePerezAltamirano sorry that this here is so complicated for you, just realized your whole youtube content is not in english. We require english content for now at least. I can keep your PR open, flag it as international and if we add more languages we could merge you but for now you either have to remove your feed or we can not merge that.

We just have to draw a line somewhere :(

**Comment by matthiasjost** [comment]
@RicardoJosuePerezAltamirano Can you remove the feeds that are not in English, so we can merge our PR anyways then?

**Pull request closed without merging**